### ISSUE TITLE
Testing find my mates

### ISSUE THREAD

**Issue by argenos** [b-it-bots/mas_domestic_robotics#143]
This one doesn't really require further description, but @PatrickNa you can add any further things that remained as todos.

**Comment by argenos** [comment]
@robertocaiwu could you help us out with running this test (maybe with the help of @henrikschnor if he has some available time during exam season?)?

The important thing out of this is finding out already things that don't work or need to be updated to work on the lab instead of GO.

**Comment by robertocaiwu** [comment]
I met with @henrikschnor  and we identified a few problems when trying to run this scenario. Since issues/ pull request have been raised, I will link them:

- [x] Speech models was not pointing to the correct directory. Solved [#195]
- [ ] NLU Rasa models for find my mates is not present. (needed for "find" intend and for names and description of persons) [b-it-bots/mas_models/issues/5]
- [ ] Find people package was not running. (Tried running the package by itself, but does not seem to return a result.)  [#152]
